src/blocksparse/combiner.jl

```diff
@@ -1,3 +1,6 @@
+#<fermions>:
+before_combiner_signs(T, labelsT, indsT, C, labelsC, indsC, labelsR, indsR) = T
+after_combiner_signs(R, labelsR, indsR, C, labelsC, indsC) = R
```

Comment:
Also maybe could be `after_combiner_signs(R, args...)`? I guess in principle this signature adds some kind of documentation and may catch errors of not inputting the correct number of inputs.